public/js/components/FormFields/SearchFields/SearchSuggestions.js

```diff
@@ -12,6 +27,8 @@ export default class SearchSuggestions extends React.Component {
   };
 
   initialState = 'idle';
+
+  store = createStore(this.transition, this.initialState);
```

Comment:
We could use the existing store. There's a block at the bottom of some of the components that create a 'connected component', that is, a component with access to the redux store on props. You can use this to pass action creators into a component and call changes to the store, which are passed back down as props.

If you're free tomorrow, I can come pair on this with you?

Reply:
Yep that'd be most welcome 😽